addons/account/views/partner_view.xml

```diff
@@ -225,9 +225,10 @@
                                        options="{'no_create': True, 'no_edit': True}"/>
                             </group>
                             <group string="Accounting Entries" name="accounting_entries" groups="account.group_account_readonly">
+                                <field name="is_coa_installed" invisible="1"/>
                                 <field name="currency_id" invisible="1"/>
-                                <field name="property_account_receivable_id" required="True"/>
-                                <field name="property_account_payable_id" required="True"/>
+                                <field name="property_account_receivable_id" required="is_coa_installed"/>
+                                <field name="property_account_payable_id" required="is_coa_installed"/>
```

Comment:
@epictete not sure if we can do that since https://github.com/odoo/odoo/commit/de302c2d36305c0d7562572a30587641eabfe914 ?